spec/controllers/concerns/laying_date_helper_spec.rb

```diff
@@ -73,18 +73,18 @@
 
     context 'work_packaged_thing has no layings' do
       it 'returns nil' do
-        date = described_class.get_date(work_package_no_layings)
+        date = described_class.get_date(laying_connected_thing_no_layings)
 
         expect(date).to be(nil)
       end
     end
 
     context 'work package has no work_packaged_thing' do
       it 'returns nil' do
-        date = described_class.get_date(work_package_no_work_packaged_thing)
+        date = described_class.get_date(laying_connected_thing_no_work_packaged_thing)
 
         expect(date).to be(nil)
       end
```

Comment:
Can we add a test to check the logic in the conditional to check it gets the date correctly for a LaidThing?

Reply:
🎉